nubi/src/components/buttons/about/styles.module.scss

```
@@ -0,0 +1,12 @@
+.button{
+  background-color: #5481ff;
+  border: none;
+  color: white;
+  border-radius: 5vh;
+  width: 20vh;
+  height: 6vh;
+  margin: 2vh;
+}
+.text{
```

Recuerda siempre dejar espacios entre las definiciones de los selectores. También recuerda los espacios entre el selector y la llave de apertura. Te dejo la [guía de estilos](https://google.github.io/styleguide/htmlcssguide.html) de CSS y HTML que usa Google para que puedas chusmear un poco cómo se acomoda el código en otras empresas.
```suggestion
}

.text {
```